doc/03_reference/testplan.rst

```diff
@@ -20,10 +20,10 @@ Testbench Architecture
 .. figure:: images/tb2.svg
     :alt: Testbench Architecture
 
-    Architecture of the UVM testbench for Ibex core
+    Architecture of the UVM testbench for CVE2 core
```

Comment:
We are not using this verification environment.  I have created task #281 to resolve.